compiler/optimizer/IsolatedStoreElimination.cpp

```diff
@@ -219,14 +212,10 @@ int32_t TR_IsolatedStoreElimination::perform()
           traceMsg(comp(), "Starting Global Store Elimination (using use/def info)\n");
       cost = performWithUseDefInfo();
       }
-   else if (!_mustUseUseDefInfo)
-      {
-      if (trace())
-          traceMsg(comp(), "Starting Global Store Elimination (without using use/def info)\n");
-      cost = performWithoutUseDefInfo();
-      }
```

Comment:
This doesn't look right.  This `else if ` path is the one you want to keep because `_mustUseUseDefInfo` is initialized to `false` just before the `if`.

Reply:
Hi @0xdaryl , I have just made the change accordingly.